src/utils/conversion.ts

```diff
@@ -3362,3 +3364,32 @@ export function agentGroupToPermissionGroup(
     }
   }
 }
+
+/**
+ * @hidden
+ */
+export async function portfolioNameToNumber(
+  rawIdentityId: IdentityId,
+  rawName: PortfolioName,
+  query: QueryableStorage<'promise'>
+): Promise<BigNumber | undefined> {
```

Comment:
I think this should be null:
```suggestion
): Promise<BigNumber | null> {
```

In JS `undefined` means not initialized, where as `null` communicates there wasn't actually a value.

I think the behavior is pretty similar though.

(test case + return would need to be updated as well)